--- a/dpi/bookmarks.c
+++ b/dpi/bookmarks.c
@@ -111,7 +111,7 @@ DOCTYPE
 "	}\n"
 "</style>\n"
 "</head>\n"
-"<body id='dillo_bm' bgcolor='#6980E0' link='black' vlink='brown'>\n"
+"<body id='dillo_bm' bgcolor='#778899' link='black' vlink='brown'>\n"
 "<table border='0' cellpadding='0' width='100%'>\n"
 " <tr><td>\n"
 "  <table width='100%' bgcolor='transparent;'>\n"
@@ -145,7 +145,7 @@ DOCTYPE
 "	}\n"
 "</style>\n"
 "</head>\n"
-"<body id='dillo_bm' bgcolor='#6980E0' link='black' vlink='brown'>\n"
+"<body id='dillo_bm' bgcolor='#778899' link='black' vlink='brown'>\n"
 "<table border='0' cellpadding='0' width='100%'>\n"
 " <tr><td>\n"
 "  <table width='100%' bgcolor='transparent;'>\n"
